tests/baselines/reference/privateNameInObjectLiteral-1.js

```diff
@@ -0,0 +1,10 @@
+//// [privateNameInObjectLiteral-1.ts]
+const obj = {
+    #foo: 1
+};
+
+
+//// [privateNameInObjectLiteral-1.js]
+var obj = {
+    : 1

```

Comment:
this emit is a little weird. I thought we usually had syntactically correct emit even in the presence of errors.

Reply:
@rbuckton suggested using empty identifiers in the case of bad syntax. Previously I was just stripping the '#' and making it an identifier.